GDV-HERBST/appendix/Glossar.md

```diff
@@ -143,20 +148,38 @@
 
 ![Prozesskette](assets/prozesskette.png)
 
+## Digitale Transformation
+
+Der fortlaufende, durch digitale Technologien begründete gesamtgesellschaftliche Veränderungsprozess.
+Betrachtet werden vor allem Auswirkungen auf Wirtschaft und Industrie.
+
 ## Digitaler Zwilling
 
 - digitale Repräsentation eines Objektes aus der realen Welt in der digitalen Welt
+- Beispiele:
+  - Track and Trace in der Logistik
+  - Anwendungen im PLM
 
 ## Dithering
 
 - Erzeugen der Illusion einer größeren Farbtiefe
 - Nachbildung von fehlenden Farben durch bestimmte Pixelanordnung
 
+## e-collaboration
+
+Beschreibt die Maßnahmen zur dezentralen computergestützten Zusammenarbeit von zeitlich und/oder räumlich getrennten Teams und Gruppen, die an einer gemeinsamen Aufgabe arbeiten.
+
+Hier: arbeiten gemeinsam an einem Produkt.
+
+> Quelle [Wikipedia: e-collaboration](https://de.wikipedia.org/wiki/E-Collaboration)
+
 ## ERP
 
-> **E**nterprise **R**esource **P**lanning
+**E**nterprise **R**essource **P**lanning beschreibt Softwarelösungen zur Ressourcenverwaltung und -planung in einem Unternehmen oder einer Organisation.
```

Comment:
```suggestion
**E**nterprise **R**esource **P**lanning beschreibt Softwarelösungen zur Ressourcenverwaltung und -planung in einem Unternehmen oder einer Organisation.
```

In einem englischen Ausdruck, würd ich auch die englische Schreibweise beibehalten.